src/uniswap-summaries.md

```diff
@@ -3439,6 +3439,40 @@ module SOLIDITY-UNISWAP-MINT-SUMMARY
 endmodule
 ```
 
+```k
+module SOLIDITY-UNISWAP-APPROVE-SUMMARY
+  imports SOLIDITY-CONFIGURATION
+  imports SOLIDITY-EXPRESSION
+  imports SOLIDITY-UNISWAP-TOKENS
+
+  syntax KItem ::= "stopFunction"
```

Comment:
This is leftover?

Reply:
Thanks for pointing this out, and sorry for forgetting to delete it!